phpcs.xml.dist

```diff
@@ -41,6 +41,7 @@
 	<exclude-pattern>/wordpress-importer/*</exclude-pattern>
 	<exclude-pattern>/wp-parsely/*</exclude-pattern>
 	<exclude-pattern>*/build/*\.asset\.php</exclude-pattern>
+	<exclude-pattern>/vip-integrations/*</exclude-pattern>
```

Comment:
Why is this being excluded?

Reply:
I had initially had this targeted at just the specific vendor folder, but changed it to the whole folder. I'll go back to just the vendor folder instead.

Reply:
Pushed it up!